patches/components-content_settings-core-common-content_settings_types.mojom.patch

```diff
@@ -23,5 +23,4 @@ index 406f182232279b9a5accaa530891de906455b1a1..1dc2cd19cf575f5b4811e523e9aa14fb
 +  BRAVE_HTTPS_UPGRADE,
 +  BRAVE_REMEMBER_1P_STORAGE,
 +  BRAVE_LOCALHOST_ACCESS,
-+  NUM_TYPES,
```

Comment:
I think this enum now can be extended via `chromium_src` override.
`chromium_src/components/content_settings/core/common/content_settings_types.mojom`
```
[BraveExtend]
enum ContentSettingsType {
  BRAVE_START,
  BRAVE_ADS = BRAVE_START,
  BRAVE_COSMETIC_FILTERING,
  BRAVE_TRACKERS,
  BRAVE_HTTP_UPGRADABLE_RESOURCES,
  BRAVE_FINGERPRINTING_V2,
  BRAVE_SHIELDS,
  BRAVE_REFERRERS,
  BRAVE_COOKIES,
  BRAVE_SPEEDREADER,
  BRAVE_ETHEREUM,
  BRAVE_SOLANA,
  BRAVE_GOOGLE_SIGN_IN,
  BRAVE_HTTPS_UPGRADE,
  BRAVE_REMEMBER_1P_STORAGE,
  BRAVE_LOCALHOST_ACCESS,
};
```